src/cryptojwt/key_bundle.py

```diff
@@ -462,21 +463,25 @@ def update(self):
                         self.do_local_jwk(self.source)
                     elif self.fileformat == "der":
                         self.do_local_der(self.source, self.keytype, self.keyusage)
+                    updated = True
```

Comment:
I think we should make `do_local_jwk` `do_local_der` and `do_remote` behave the same - have the same signature and return type. I can definitely see cases where the "local" operations can fail - this should not be ignored. So `updated` should get its value the same way it gets is from `do_remote`, ie from the return value of the `do_local_...` operations.